micronota/util.py

```diff
@@ -102,9 +115,9 @@ def _download(src, dest, **kwargs):
 
 @contextmanager
 def _tmp_file(*args, **kwargs):
```

Comment:
because the file will be deleted immediately when it is closed, which seemed a deal breaker. let me rethink about it.